src/game/room.c

```diff
@@ -536,11 +453,75 @@ int16_t Room_GetHeight(
     return height;
 }
 
+static int16_t Room_GetFloorTiltHeight(
+    const SECTOR_INFO *sector, const int32_t x, const int32_t z)
+{
+    int16_t height = sector->floor.height;
+    if (sector->floor.tilt == 0) {
+        return height;
+    }
+
+    const int32_t z_off = sector->floor.tilt >> 8;
```

Comment:
We mentioned getting rid of shifts, but it looks like it's still used here?

`why do we shift left by 8? should this be * STEP_L?`

`Yeah, it's moving it into world units. Thinking about it, should we just do this once on level load too? It seems (almost) every reference to it does the shift, so we could just store it as an int16_t and do away with all the shifting`

Reply:
We could store X/Z tilts as `int8_t` in the sector. `Room_GetTiltType` expects the `int16_t` though but I'll see what can be done there.